companion/src/CMakeLists.txt

```diff
@@ -118,7 +118,15 @@ add_subdirectory(shared)
 add_subdirectory(modeledit)
 add_subdirectory(generaledit)
 add_subdirectory(simulation)
-add_subdirectory(firmwares/opentx/simulator)
+
+SET (translation FR IT DE EN)
```

Comment:
No this is not what I would like to see. I would prefer on demand compilation, just as we do for firmwares (you will download your simu)